src/utils/api.js

```diff
@@ -28,4 +27,18 @@ export const fetchComments = (review_id) => {
     .then(({ data }) => {
         return data.comments
     })
-}
\ No newline at end of file
+}
+
+export const addVotes = (review_id, setCurrentVotes, setButtonDisable, setVoteCastText) => {
+    const increment = { inc_votes: 1 };
+    return reviews
+    .patch(`/reviews/${review_id}`, increment)
+    .then(({ data }) => {
+        setCurrentVotes(data.votes);
+        setButtonDisable(true);
+        setVoteCastText('Your vote has been added!')
+    })
+    .catch((error) => {
+        setVoteCastText('Error, please try again!')
```

Comment:
Your api file shouldnt handle the setting of state, you should just use this to complete the request to the db. 

Any state setting should happen in the react component. 

You also need to update the votes optimisitically. At the moment, on a slow connection, I click the button and nothing happens for a while. I could then click it multiple times, thinking its not working and end up doing multiple requests